ps_facetedsearch.php

```diff
@@ -729,8 +729,12 @@ public function getContent()
         $this->context->smarty->assign('uri', $this->getPathUri());
 
         // Assign assets
-        if (method_exists($this->context->controller, 'addJquery')) {
-            $this->context->controller->addJS(_PS_JS_DIR_ . 'jquery/plugins/jquery.sortable.js');
+        if (file_exists(_PS_ROOT_DIR_ . _PS_JS_DIR_ . 'vendor/Sortable.min.js')) {
+            $this->context->controller->addJS(_PS_JS_DIR_ . 'vendor/Sortable.min.js');
+        } else {
```

Comment:
Not blocking but you should add an `elseif` instead of `if` in `else`

Reply:
@lartist Nothing change when replace `else` by `elseif` but it will be reasonable if you can provide a document that declares we `should` use `elseif` in this case.

Reply:
https://www.perforce.com/blog/qac/what-cyclomatic-complexity :)

Reply:
The link about CYC is too complex, and I can't see a direct relation with this case.
The common rule in this module and PHP manual is `elseif` must follow by at least one `else`
1. [ps_facetedsearch.php#L297-L304](https://github.com/PrestaShop/ps_facetedsearch/blob/dev/ps_facetedsearch.php#L297-L304)
2. [ps_facetedsearch.php#L591-L595](https://github.com/PrestaShop/ps_facetedsearch/blob/dev/ps_facetedsearch.php#L591-L595)
3. https://www.php.net/manual/en/control-structures.elseif.php

Reply:
In this case we can use `else if` to save two curly braces but it breaks the consistency, and make code less readable :)